### ISSUE TITLE
32 bit gst-plugins-base development files not found?

### ISSUE THREAD

**Issue by mikeserv-git** [Frogging-Family/wine-tkg-git#992]
A proton-tkg build, valve-exp-bleeding-edge, current wine-tkg-git sources from a few minutes ago. (Proton_8.0)

I got the build off the ground, it was building nicely so I thought, but during the wine32 configuration,

checking for gst/gst.h... no
configure: error: gstreamer-1.0 base plugins 32-bit development files not found, GStreamer won't be supported.
This is an error since --with-gstreamer was requested.

I found the 32 bit config.log, it really isn't finding the header in context (i.e. not some other compile test failure causing the no result)

configure:17092: checking for gst/gst.h
configure:17092: gcc -m32 -c -O2 -ftree-vectorize   conftest.c >&5
conftest.c:179:10: fatal error: gst/gst.h: No such file or directory
  179 | #include <gst/gst.h>
      |          ^~~~~~~~~~~
compilation terminated.

There was no problem with the wine64 build finding the headers.

I don't think there are a separate set of includes for 32 bit gstreamer (not in Arch, anyway) it's just not finding it in the environment. As you can see there ought to be some -I/usr/include dirs and -l library linking flags in that compile test command.

I tried exporting PKG_CONFIG_PATH=/usr/lib/pkgconfig:/usr/lib32/pkgconfig but that also didn't help.

I can build a complete Wine, (wine64 and wine32) --with-gstreamer outside of this build.

Of course, if I figure out a solution I'll be all fingers and thumbs sharing it but at the moment I'm still stumped :-)

**Comment by mikeserv-git** [comment]
It's definitely getting the PKG_CONFIG_PATH='/usr/lib32/pkgconfig' variable, I even tried forcing it in wine-tkg-scripts/build-32.sh right in front of the configure command (although that is being set above in the script anyway). That should have no choice.

That works on my system, I tested setting variables etc. and also as I said I did a wine build to test. wine-tkg 8.6.1 (I use Kron4ek's wine-tkg sources for my system wine) and had no problems with the wine32 gstreamer using PKG_CONFIG_PATH

It could be a broken configure script or something in Valve's wine. (I looked at it, but came up short)

**Comment by Tk-Glitch** [comment]
Pushed a workaround with https://github.com/Frogging-Family/wine-tkg-git/commit/a24a407563a11a40680ffe694088b52a202fa5ab

**Comment by mikeserv-git** [comment]
I hope it doesn't annoy, posting comments to closed issues, but I just want to let you know, that with the things you have done today (well, up to f742de3 ) that I got a successful build of

proton_tkg_experimental.bleeding.edge.8.0.41693.20230425

AND the protonmediaconverter works. I tested Borderlands 3, if that works there, it works for me.

I'll have to put this build through its paces tonight and test a lot of other games, but that's a very good start.

So thanks for everything.
